src/DNS_Mgr.h

```diff
@@ -2,6 +2,7 @@
 
 #pragma once
 
+#include <ares.h>
```

Comment:
It does need `ares.h` because there's an `ares_channel` environment variable. Unfortunately, `ares_channel` isn't a structure we can just forward declare, it's a pointer to a structure. I can redefine it the way that they do internally but that seems somewhat brittle (it's a forward-declared structure, then a typedef to a pointer to that).

Reply:
So then we are installing the c-areas headers as well? Is that robust against c-ares existing on the system already somehow, potentially in a different version?